client/src/pages/DetailedGasStationPage.jsx

```diff
@@ -17,12 +17,12 @@ function DetailedGasStationPage({ title, fuelValue, toggleFavorite, favoriteIDs
         Authorization: `Bearer ${stationsApiKey}`,
       },
     })
-    
       .then(response => response.json())
       .then(json => setCurrentStation(json))
       .catch(error => {
         console.log('There has been a problem with your fetch operation:', error);
```

Comment:
Not important, but a nice point:
```suggestion
        console.error('There has been a problem with your fetch operation:', error);
```

Reply:
Thanks for the tip.